gitlab/__init__.py

```diff
@@ -355,7 +355,7 @@ def _set_auth_info(self):
                 bool(arg)
                 for arg in [self.private_token, self.oauth_token, self.job_token]
             )
-            != 1
+            > 1
```

Comment:
But this does exactly the reverse. It only works, if we have multiple parameters.

We should only check this condition, if the items are larger than 1